pkg/ottl/contexts/ottlspanevents/span_events.go

```diff
@@ -0,0 +1,177 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ottlspanevents // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottlspanevents"
```

Comment:
Should we use `ottlspanevent` instead? True for other places like the `datapoint`?

Reply:
Ya we aren't consistent.  I agree singular is better.  I will get this PR updated and then update other contexts in another PR.  Also, I think we should rename `ottltraces` to `ottlspan`.